### DISCUSSION TITLE
Expose pdf sections/subsections to link copy template

### DISCUSSION THREAD

**Discussion by tommyhosman** [RyotaUshio/obsidian-pdf-plus#175] [Q&A]
Great tool!

When I am organizing/reviewing my highlights and notes, I keep wanting to also see the pdf section/subsection alongside the highlighted text. Do you know if this is something that is easy to extract and expose?

Additionally, it'd be fantastic to have the Auto-Paste be pdf section aware. That is, create md headers for the relevant sections and paste the highlight/selection at the end of the respective header section (or subsection). I can see it being useful to specify the max depth of section/subsections the user want.

Regardless, thanks again for this great viewer!

**Comment by N3C2L** [comment]
I don't have an answer to your request, but this information could be useful for new users who are searching for something similar: There is a very cool shortcut/command which you can assign/use to extract your headings from the PDF file: "Copy PDF outline as markdown headings". You can paste that headings to your markdown annotation file and have these headings then visible inside your native obsidian outline sidebar. You can collapse and expand these headings inside your markdown file and auto-paste where you want.
